src/validate/mod.rs

```diff
@@ -11,14 +11,16 @@ mod test;
 lazy_static! {
     static ref AWS_ACCESS_FORMAT: Regex = Regex::new("^[A-Z0-9]+$").unwrap();
     static ref AWS_SECRET_FORMAT: Regex = Regex::new("^[A-Za-z0-9+/=]+$").unwrap();
-    static ref HOST_FORMAT: Regex = Regex::new("^[A-Za-z0-9-]+(?:\\.[A-Za-z0-9-]+)*$").unwrap();
-    static ref PROVIDER_FORMAT: Regex = Regex::new("^(?:mock|ses|smtp)$").unwrap();
-    static ref SENDER_FORMAT: Regex = Regex::new(
-        "^[A-Za-z0-9-]+(?: [A-Za-z0-9-]+)* <[a-z0-9-]+@[a-z0-9-]+(?:\\.[a-z0-9-]+)+>$"
-    ).unwrap();
     static ref BASE_URI_FORMAT: Regex = Regex::new(
         "^https?://[A-Za-z0-9-]+(?:\\.[A-Za-z0-9-]+)*(?::[0-9]+)?/(?:[A-Za-z0-9-]+/)*$"
     ).unwrap();
+    static ref EMAIL_ADDRESS_FORMAT: Regex =
+        Regex::new("^[a-z0-9-]+@[a-z0-9-]+(?:\\.[a-z0-9-]+)+$").unwrap();
+    static ref HOST_FORMAT: Regex = Regex::new("^[A-Za-z0-9-]+(?:\\.[A-Za-z0-9-]+)*$").unwrap();
+    static ref PROVIDER_FORMAT: Regex = Regex::new("^(?:mock|sendgrid|ses|smtp)$").unwrap();
+    static ref SENDER_NAME_FORMAT: Regex =
+        Regex::new("^[A-Za-z0-9-]+(?: [A-Za-z0-9-]+)*$").unwrap();
+    static ref SENDGRID_API_KEY_FORMAT: Regex = Regex::new("^[A-Za-z0-9._=]+$").unwrap();
```

Comment:
Noticed that the API key length is 69 characters, which could be confused for the API Key ID (length 22).

Reply:
Okay I'll update, thanks.